### DISCUSSION TITLE
Whisper allucination or no-speech problem?

### DISCUSSION THREAD

**Discussion by emanueleielo** [openai/whisper#1308] [Q&A]
Hi guys, recently I transcribed a audio of 1hour, but in the first 2 minutes the transcription start to be ‘.. . . … … .. .. …’ and next stop to transcribing.

In that part of audio there is another speaker that is talking with the principal one with a lower voice.

Actually I would like to ask where I should work for solve this kind of problem is it a
problem of allucination or something like no-speech?  

I expect that when the model can’t hear what the speakers saying is just skip that part and keep going.

Thank u!

**Comment by phineas-pta** [comment]
seem like hallucination, it's discussed here #679

TLDR: use SileroVAD to remove silence
